learn/parse_http.go

```diff
@@ -288,6 +288,17 @@ func parseBody(contentType string, bodyStream io.Reader, statusCode int) (*pb.Da
 		return nil, errors.Wrapf(err, "failed to parse MIME from Content-Type %q", contentType)
 	}
 
+	// Rewrite media type to JSON for types encoded as JSON
+	// TODO: XML parsing
+	// TODO: application/json-seq (RFC 7466)?
+	// TODO: more text/* types
+	switch {
+	case strings.HasSuffix(mediaType, "+json"):
+		mediaType = "application/json"
+	case mediaType == "text/csv":
+		mediaType = "text/plain"
```

Comment:
Why not add a follow-through case for `text/csv` in the `switch` below?

Reply:
Yeah... that would probably be better for this particular case, I originally tried to handle

strings.HasPrefix(mediaType, "text/")

and then edited it just down to this.